tests/markets/conftest.py

```diff
@@ -165,12 +166,12 @@ def feed_infos():
     depth_path = '../../feeds/univ3_axs_weth'
 
     raw_uni_framed_market_path = os.path.join(base,
-                                              market_path +
-                                              '_raw_uni_framed.json')
+                                              market_path
+                                              + '_raw_uni_framed.json')
```

Comment:
Why? Changes like this exaggerate pull requests and might lead to annoying conflicts? Just leave it the way it was in the future